medicines/search/Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: get-env-dev
+get-env-dev: ## run locally
```

Comment:
See https://github.com/MHRA/products/pull/604/files  where I fixed all the `make help` outputs